sound/pulseaudio/Makefile

```diff
@@ -157,140 +148,62 @@ endif
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) \
-		$(1)/usr/lib/pkgconfig \
-		$(1)/usr/include/pulse \
-		$(1)/usr/lib \
-		$(1)/usr/lib/pulseaudio
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/pulse/* \
-		$(1)/usr/include/pulse
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/*.pc \
-		$(1)/usr/lib/pkgconfig
-	$(SED) \
-		's,/usr/include,$$$${prefix}/include,g' \
-		$(1)/usr/lib/pkgconfig/libpulse.pc
-	$(SED) \
-		's,/usr/lib,$$$${exec_prefix}/lib,g' \
-		$(1)/usr/lib/pkgconfig/libpulse.pc
-	$(SED) \
-		's,/usr/include,$$$${prefix}/include,g' \
-		$(1)/usr/lib/pkgconfig/libpulse-simple.pc
-	$(SED) \
-		's,/usr/lib,$$$${exec_prefix}/lib,g' \
-		$(1)/usr/lib/pkgconfig/libpulse-simple.pc
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/*.so* \
-		$(1)/usr/lib/
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/pulseaudio/* \
-		$(1)/usr/lib/pulseaudio/
+	$(INSTALL_DIR) $(1)/usr/include/pulse
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/pulse/* $(1)/usr/include/pulse
+
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/*.pc $(1)/usr/lib/pkgconfig
+	$(SED) 's,/usr/include,$$$${prefix}/include,g' $(1)/usr/lib/pkgconfig/libpulse.pc
```

Comment:
Something tells me this is outdated…

Reply:
I agree with you, but I would rather not touch or modify it in this PR since it was there and avoid doing more changes than necessary right now. I need to rework commit messages, hopefully, I will do that tomorrow and then after merging this, we can look at it further. Those three lines things were really hard to follow and compare.